mbed-cli/create-module.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -eu
+
+
+
+MBED_APP='mbed-os-example-tls'
+
+# make sure we're running in our own directory
+if [ -f create-module.sh ]; then :; else
+    cd $( dirname $0 )
+    if [ -f create-module.sh ]; then :; else
+        echo "Please run the script from is directory." >&2
+        exit 1
+    fi
+fi
+
+REMOTE_URL=$(dirname `git remote -v | grep fetch | grep ARMmbed | awk '{print $2}'`)
```

Comment:
I think you are trying to extract the URL of the remote to clone. I am not sure this works correctly if you have multiple remotes both in ARMmbed (which has happened in the past). Perhaps is better to just hard code the URL string or pass it in the command line arguments to the script?